00-strings/test_strings.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import strings
+
+class TestStrings(unittest.TestCase):
+
+    def test_get_string_length(self):
+        self.assertEqual(strings.length(''), 0)
+        self.assertEqual(strings.length('foo'), 3)
+
+    def test_get_string_without(self):
+        # implement strings.without that will remove given characters
+        self.assertEqual(strings.without('abba', 'a'), 'bb')
+        self.assertEqual(strings.without('abba', 'cd'), 'abba')
+        self.assertEqual(strings.without('foobar', 'ob'), 'far')
+
+    def test_count_discint_chars(self):
+        # implement strings.count_distinct that will count the number of distinct characters
+        self.assertEqual(strings.count_distinct('abba'), 2)
+        self.assertEqual(strings.count_distinct('abcdfda'), 5)
+
+    def test_tokens(self):
+        # implement strings.tokens that will return a list of substrings split from . and removes empty tokens
+        self.assertEqual(strings.tokens('foo.bar'), ['foo', 'bar'])
+        self.assertEqual(strings.tokens('..foo..bar..'), ['foo', 'bar'])
```

Comment:
The exercise doesn't really specify this as expected behaviour. The exercise says it should return "a list of tokens in a string". I'm not entirely sure, but is "token" a universally understandable word, which means "characters separated by a dot (`.`) and length must be over 0"?

Reply:
Changed the documentation